ftwhttp/header_test.go

```diff
@@ -104,20 +104,54 @@ func (s *headerTestSuite) TestHeaderWriteString() {
 	}
 }
 
-func (s *headerTestSuite) TestHeaderSetGet() {
+func (s *headerTestSuite) TestHeaderAdd() {
 	h := Header{
 		"Custom": "Value",
 	}
 	h.Add("Other", "Value")
-	value := h.Get("Other")
+	value := h["Other"]
```

Comment:
Use `Get` for access in all tests. You shouldn't test the internal representation.

Reply:
We both agreed that a unit test should check one method to easily identify what failed if the test fails. Do we want to strictly adhere to this principle or not?  For me we do not test the internal representation, we verify if `Add`/`Get` worked as expected. I am eager to know your opinion about that.

Reply:
By using `h["X"]`, you are verifying the internal representation because you rely on the knowledge that the canonical form of `x` is `X`. In theory, we might want to change the internal representation, without changing the API, then these tests will break.